src/components/ModuleStudents/ModuleStudents.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Card from 'components/Card/Card';
+import { ReactComponent as LockIcon } from 'routes/modules/lock.svg';
+import { CircularProgressBar } from 'components/CircularProgressBar/CircularProgressBar';
+import { StudentModule } from 'types/Module/Module';
+
+import style from './ModuleStudents.module.css';
+
+type Props = {
+  data: StudentModule[];
+  cardStyles: (index: number) => string;
+};
+
+export function ModuleStudents({ data, cardStyles }: Props) {
+  return (
+    <div className={style.modules}>
+      {data.map((module, i) => (
```

Comment:
Just as note for future us: We need to check with the backend team that the modules will be given in order. Otherwise our view logic might break.